angrmanagement/ui/menus/view_menu.py

```diff
@@ -75,9 +75,9 @@ def __init__(self, main_window):
         self.entries.extend([
             ToolbarMenu(main_window),
             MenuSeparator(),
-            MenuEntry('Next Tab', main_window.workspace.view_manager.next_tab, shortcut=QKeySequence("Ctrl+Tab")),
+            MenuEntry('Next Tab', main_window.workspace.view_manager.next_tab, shortcut=QKeySequence("Alt+Tab")),
```

Comment:
We should avoid re-binding Alt-Tab because it's already used for global window switching

Reply:
Would you prefer to have this one line reverted, just having `Alt[0-9]` remain? Edit the global switching to something else? Or Close this PR @mborgerson ?